tools/sctool/src/main/java/com/horizen/ScBootstrappingToolCommandProcessor.java

```diff
@@ -624,7 +624,8 @@ private void processEncodeString(JsonNode json ) {
             return;
         }
         String toEncode = json.get("string").asText();
-        String encoded = BCrypt.hashpw(toEncode, BCrypt.gensalt());
+        String encoded = BCrypt.with(BCrypt.Version.VERSION_2Y).hashToString(12, toEncode.toCharArray());
```

Comment:
Consider the performance of the algorithm for the factor 12.
Discuss with Cronic the suitable value of `factor` and ask if we need to let user choose the factor or/and the salt values

Reply:
I've talked with cronic and we agreed to use 8 as a algorithm cost factor